frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -194,9 +194,18 @@ public void refreshModel() {
   }
 
   @UIEffect
-  private void setTextForEmptyTable(String upperText, String lowerText, @UI Runnable onClickRunnableAction) {
+  private void setTextForEmptyTable(String upperText, @Nullable String lowerText, @Nullable @UI Runnable onClickRunnable) {
     var attrs = new SimpleTextAttributes(SimpleTextAttributes.STYLE_PLAIN, JBUI.CurrentTheme.Link.linkColor());
-    getEmptyText().setText(upperText).appendSecondaryText(lowerText, attrs, /* listener */ __ -> onClickRunnableAction.run());
+    var statusText = getEmptyText().setText(upperText);
+    if (lowerText != null) {
+      statusText.appendSecondaryText(lowerText, attrs,
+          /* listener */ onClickRunnable != null ? __ -> onClickRunnable.run() : null);
```

Comment:
Hmmm is this lambda really needed? We can't just  pass runnable here, right?

Reply:
No, lambda is needed because `appendSecondaryText` need `ActionListener` as third argument (not `Runnable`) but thanks to `ActionListener` is a functional interface we can simply wrap our `Runnable` with lambda and then pass to `appendSecondaryText`